### PR TITLE
Allow user to use rank name as well as rank number in rank & role binding

### PR DESCRIPTION
**Pull request by OnlyTwentyCharacters**
binding.rank can now either be rank number or rank name.

### CODE REVIEW

**Comment by OnlyTwentyCharacters** [comment]
https://github.com/evaera/RoVer/issues/17

File: src/DiscordBot.js [hunk 1]
```diff
@@ -178,7 +178,7 @@ class DiscordBot {
                     break;
                 case "!roverbindgrouprank":
                     if (argument.length > 0) {
-                        let bindArgs = argument.split(':');
+                        let bindArgs = argument.split(' ');
```
Comment (devSparkle) [inline src/DiscordBot.js]: Why has the split string been changed?

Comment (evaera) [inline src/DiscordBot.js]: I purposefully chose `:` as a delimiter because many role names use spaces. See #17 for my solution i'm working on

**Comment by evaera** [comment]
Thanks for the pull request. I've fixed this issue with a different approach that I think will be better for long term. (:

**Pull request closed without merging**